### ISSUE TITLE
Recursion Error with all_the_strategies on Av(12543)

### ISSUE THREAD

**Issue by jaypantone** [PermutaTriangle/Tilings#394]
Found in about three hours using the forest hammer, will follow up with more info later. Used the TrackedSearcher, and the RuleDBForest.

```python
error with class:  01432
01432
all_the_strategies_1_tracked_fusion_req_corrob_expand_verified_short_obs_4
Looking for recursive combinatorial specification with the strategies:
Inferral: row and column separation, obstruction transitivity
Initial: rearrange assumptions, add assumptions, factor, tracked fusion, requirement corroboration
Verification: verify atoms, insertion encoding verified, one by one verification, locally factorable verification, short (length <= 4) crossing obstruction verification
Set 1: point insertion, all placements

Traceback (most recent call last):
  File "/users/personnel/jay/atrap/repos/handy-scripts/jay/forest-hammer/forest-hammer.py", line 183, in run_work_packet
    spec = css.auto_search(perc=10)
  File "/users/personnel/jay/atrap/repos/comb_spec_searcher/comb_spec_searcher/comb_spec_searcher.py", line 496, in auto_search
    spec_rules = self._auto_search_rules(**kwargs)
  File "/users/personnel/jay/atrap/repos/comb_spec_searcher/comb_spec_searcher/comb_spec_searcher.py", line 529, in _auto_search_rules
    expansion_time, status_update, status_start, auto_search_start
  File "/users/personnel/jay/atrap/repos/comb_spec_searcher/comb_spec_searcher/comb_spec_searcher.py", line 578, in _expand_classes_for
    self._expand(comb_class, label, strategies, inferral)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 115, in _expand
    self.add_rule(start_label, end_labels, rule)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 254, in add_rule
    self._expand(assumption_tiling, label, (rule.strategy,), False)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 115, in _expand
    self.add_rule(start_label, end_labels, rule)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 226, in add_rule
    self._expand(comb_class, child_label, old_strategies, False)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 115, in _expand
    self.add_rule(start_label, end_labels, rule)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 230, in add_rule
    self._expand(comb_class, child_label, self.tracking_strategies, False)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 115, in _expand
    self.add_rule(start_label, end_labels, rule)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 226, in add_rule
    self._expand(comb_class, child_label, old_strategies, False)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 115, in _expand
    self.add_rule(start_label, end_labels, rule)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 226, in add_rule
    self._expand(comb_class, child_label, old_strategies, False)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 115, in _expand
    self.add_rule(start_label, end_labels, rule)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 226, in add_rule
    self._expand(comb_class, child_label, old_strategies, False)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 115, in _expand
    self.add_rule(start_label, end_labels, rule)

  [4000 lines omitted]
  
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 226, in add_rule
    self._expand(comb_class, child_label, old_strategies, False)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 115, in _expand
    self.add_rule(start_label, end_labels, rule)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 226, in add_rule
    self._expand(comb_class, child_label, old_strategies, False)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 115, in _expand
    self.add_rule(start_label, end_labels, rule)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 226, in add_rule
    self._expand(comb_class, child_label, old_strategies, False)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/tilescope.py", line 109, in _expand
    comb_class, strategy_generator, label
  File "/users/personnel/jay/atrap/repos/comb_spec_searcher/comb_spec_searcher/utils.py", line 62, in inner
    for res in func(css, *args, **kwargs):
  File "/users/personnel/jay/atrap/repos/comb_spec_searcher/comb_spec_searcher/comb_spec_searcher.py", line 221, in _expand_class_with_strategy
    for rule in self._rules_from_strategy(comb_class, strategy_generator):
  File "/users/personnel/jay/atrap/repos/comb_spec_searcher/comb_spec_searcher/comb_spec_searcher.py", line 178, in _rules_from_strategy
    yield strategy(comb_class)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/strategies/fusion/fusion.py", line 150, in __call__
    children = self.decomposition_function(comb_class)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/strategies/fusion/fusion.py", line 162, in decomposition_function
    if algo.fusable():
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/algorithms/fusion.py", line 368, in fusable
    for counter in self.requirements_fuse_counters
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/algorithms/fusion.py", line 227, in requirements_fuse_counters
    for req_list in self._tiling.requirements
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/algorithms/fusion.py", line 228, in <listcomp>
    if not self.is_positive_left_or_right_requirement(req_list)
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/algorithms/fusion.py", line 203, in is_positive_left_or_right_requirement
    if requirement == left:
  File "/users/personnel/jay/atrap/repos/Tilings/tilings/griddedperm.py", line 686, in __eq__
    return self._patt == other.patt and self._pos == other.pos
RecursionError: maximum recursion depth exceeded
```

**Comment by jaypantone** [comment]
Currently running again with this script, which prints way too much:
```python
from tilings.tilescope import TileScopePack, TrackedSearcher
from comb_spec_searcher.rule_db import RuleDBForest
from tilings.strategies import RequirementCorroborationFactory, ShortObstructionVerificationStrategy


basis = "01432"
pack = TileScopePack.all_the_strategies(1).make_fusion(tracked=True).add_initial(RequirementCorroborationFactory(ignore_parent=True)).add_verification(ShortObstructionVerificationStrategy(4))
ruledb = RuleDBForest()
css = TrackedSearcher(basis, pack, ruledb=ruledb, max_assumptions=10, expand_verified=True)
spec = css.auto_search(status_update=1800)
```

**Comment by jaypantone** [comment]
Interestingly, the printing paused for a long long time (many minutes) on this:
```
about to call _expand on tilings below, with rule fuse rows 2 and 3
+-+
|●|
+-+
|/|
+-+
|3|
+-+
|3|
+-+
|2|
+-+
|1|
+-+
/: Av(10)
1: Av(01432)
2: Av(0321)
3: Av(210)
●: point
Crossing obstructions:
10: (0, 5), (0, 4)
210: (0, 3), (0, 2), (0, 2)
210: (0, 3), (0, 3), (0, 2)
210: (0, 4), (0, 2), (0, 2)
210: (0, 4), (0, 3), (0, 2)
210: (0, 4), (0, 3), (0, 3)
210: (0, 5), (0, 2), (0, 2)
210: (0, 5), (0, 3), (0, 2)
210: (0, 5), (0, 3), (0, 3)
0321: (0, 1), (0, 2), (0, 1), (0, 1)
0321: (0, 1), (0, 2), (0, 2), (0, 1)
0321: (0, 1), (0, 3), (0, 1), (0, 1)
0321: (0, 1), (0, 3), (0, 2), (0, 1)
0321: (0, 1), (0, 3), (0, 3), (0, 1)
0321: (0, 1), (0, 4), (0, 1), (0, 1)
0321: (0, 1), (0, 4), (0, 2), (0, 1)
0321: (0, 1), (0, 4), (0, 3), (0, 1)
0321: (0, 1), (0, 5), (0, 1), (0, 1)
0321: (0, 1), (0, 5), (0, 2), (0, 1)
0321: (0, 1), (0, 5), (0, 3), (0, 1)
01432: (0, 0), (0, 0), (0, 1), (0, 0), (0, 0)
01432: (0, 0), (0, 0), (0, 1), (0, 1), (0, 0)
01432: (0, 0), (0, 0), (0, 1), (0, 1), (0, 1)
01432: (0, 0), (0, 0), (0, 2), (0, 0), (0, 0)
01432: (0, 0), (0, 0), (0, 2), (0, 1), (0, 0)
01432: (0, 0), (0, 0), (0, 2), (0, 1), (0, 1)
01432: (0, 0), (0, 0), (0, 2), (0, 2), (0, 0)
01432: (0, 0), (0, 0), (0, 2), (0, 2), (0, 1)
01432: (0, 0), (0, 0), (0, 3), (0, 0), (0, 0)
01432: (0, 0), (0, 0), (0, 3), (0, 1), (0, 0)
01432: (0, 0), (0, 0), (0, 3), (0, 1), (0, 1)
01432: (0, 0), (0, 0), (0, 3), (0, 2), (0, 0)
01432: (0, 0), (0, 0), (0, 3), (0, 2), (0, 1)
01432: (0, 0), (0, 0), (0, 3), (0, 3), (0, 0)
01432: (0, 0), (0, 0), (0, 3), (0, 3), (0, 1)
01432: (0, 0), (0, 0), (0, 4), (0, 0), (0, 0)
01432: (0, 0), (0, 0), (0, 4), (0, 1), (0, 0)
01432: (0, 0), (0, 0), (0, 4), (0, 1), (0, 1)
01432: (0, 0), (0, 0), (0, 4), (0, 2), (0, 0)
01432: (0, 0), (0, 0), (0, 4), (0, 2), (0, 1)
01432: (0, 0), (0, 0), (0, 4), (0, 3), (0, 0)
01432: (0, 0), (0, 0), (0, 4), (0, 3), (0, 1)
01432: (0, 0), (0, 0), (0, 5), (0, 0), (0, 0)
01432: (0, 0), (0, 0), (0, 5), (0, 1), (0, 0)
01432: (0, 0), (0, 0), (0, 5), (0, 1), (0, 1)
01432: (0, 0), (0, 0), (0, 5), (0, 2), (0, 0)
01432: (0, 0), (0, 0), (0, 5), (0, 2), (0, 1)
01432: (0, 0), (0, 0), (0, 5), (0, 3), (0, 0)
01432: (0, 0), (0, 0), (0, 5), (0, 3), (0, 1)
Requirement 0:
0: (0, 5)
Assumption 0:
can count points in cells (0, 1), (0, 2), (0, 3), (0, 4)
Assumption 1:
can count points in cells (0, 1), (0, 2), (0, 3), (0, 4), (0, 5)
```
Could this be a long is_empty check? Or maybe something else?

**Comment by jaypantone** [comment]
I think I crashed the interpreter, but the last thing that printed was this tiling / strategy combo many many times.

**Comment by jaypantone** [comment]
```
Queue status (currently on level 5):
    Queue               Size
    ---------------  -------
    working          806,254
    current (set 1)   39,995
    next                 270
        The size of the current queues at each level: 1, 11, 111, 2091, 40012
```

**Comment by jaypantone** [comment]
```
CSS status:
        Total time accounted for: 9:12:07
                                                               Number of
                                                             applications    Time spent    Percentage
    -----------------------------------------------------  --------------  ------------  ------------
    rearrange assumptions                                         742,393       3:48:12           41%
    add assumptions                                               742,394       3:11:56           34%
    requirement placement strategy                                284,400       1:04:06           11%
    requirement insertion                                          95,534       0:35:51            6%
    row and column separation                                     139,001       0:18:56            3%
    factor                                                        228,339       0:05:44            1%
   ...
```

**Comment by jaypantone** [comment]
If you add `debug=True` it crashes in under a minute. What the heck??

**Comment by jaypantone** [comment]
Got the same problem with a different 1x5 class.

**Comment by jaypantone** [comment]
Stuck on this update for hours and hours:
```
[I 211120 15:51:08 comb_spec_searcher:468] Time taken so far: 2:06:55
    CSS status:
        Total time accounted for: 1:31:27
                                                                   Number of
                                                                 applications    Time spent    Percentage
        -----------------------------------------------------  --------------  ------------  ------------
        add assumptions                                               352,542       0:26:20           28%
        requirement placement strategy                                138,392       0:23:24           25%
        rearrange assumptions                                         352,542       0:19:41           21%
        requirement insertion                                          46,547       0:07:43            8%
        row and column separation                                      82,933       0:06:14            6%
        all placements                                                  1,175       0:03:11            3%
        factor                                                        121,852       0:01:49            1%
        requirement corroboration                                      24,999       0:01:45            1%
        tracked fusion                                                 24,999       0:00:20            0%
        verify atoms                                                   48,637       0:00:07            0%
        obstruction transitivity                                       31,155       0:00:07            0%
        fuse rows 3 and 4                                               2,865       0:00:06            0%
        short (length <= 4) crossing obstruction verification          54,158       0:00:04            0%
        point insertion                                                 1,175       0:00:04            0%
        fuse rows 1 and 2                                               2,367       0:00:04            0%
        fuse columns 1 and 2                                            2,339       0:00:04            0%
        fuse columns 2 and 3                                            2,409       0:00:04            0%
        fuse rows 2 and 3                                               2,436       0:00:04            0%
        locally factorable verification                                48,631       0:00:02            0%
        fuse columns 3 and 4                                            1,045       0:00:01            0%
        insertion encoding verified                                    48,646       0:00:00            0%
        One by one subclass of Av(01432)                               48,617       0:00:00            0%
        has specification                                              38,723       0:00:00            0%
        status                                                              2       0:00:00            0%
        fuse rows 0 and 1                                                   4       0:00:00            0%
        fuse columns 0 and 1                                                4       0:00:00            0%
    ClassDB status:
        Total number of combinatorial classes found is 500,378
        is_empty check applied 35547 time. Time spent: 0:00:04
    Queue status (currently on level 5):
        Queue               Size
        ---------------  -------
        working          449,272
        current (set 1)   39,958
        next                 232
        The size of the current queues at each level: 1, 11, 111, 2087, 39976
    RuleDB status:
        Added from 2050268 normal rules
        Time spent computing forest keys: 0:25:19
        Time spent running the table method: 0:01:04
        Size of the gap: 3
        Size of the stable subset: 8922
        Sizes of the pre-images: [205039, 171758, 69871, 34487, 9141, 1160]

    Memory Status:
        ---------------------------------  ----------
        OS Allocated                        3.619 GiB
        Current Memory Used                2882.8 MiB
        Current Memory Allocated           3422.4 MiB
        Current JIT Memory Used              36.6 MiB
        Current JIT Memory Allocated         39.0 MiB
        Peak Memory Used                   3480.1 MiB
        Peak Memory Allocated Memory Used  3546.0 MiB
        ---------------------------------  ----------
        Total Garbage Collection Time: 0:13:40
     -- status update took 0.02 seconds --
```

**Comment by jaypantone** [comment]
Printing status updates every 5 minutes:

```
[I 211122 11:21:33 comb_spec_searcher:468] Time taken so far: 0:35:23
    CSS status:
        Total time accounted for: 0:24:11
                                                                   Number of
                                                                 applications    Time spent    Percentage
        -----------------------------------------------------  --------------  ------------  ------------
        requirement placement strategy                                 96,112       0:07:50           32%
        add assumptions                                               267,425       0:04:35           18%
        requirement insertion                                          46,484       0:03:58           16%
        all placements                                                  1,181       0:01:50            7%
        row and column separation                                      66,192       0:01:45            7%
        rearrange assumptions                                         267,425       0:01:45            7%
        requirement corroboration                                      24,994       0:00:56            3%
        factor                                                        103,814       0:00:49            3%
        tracked fusion                                                 24,994       0:00:12            0%
        verify atoms                                                   48,677       0:00:04            0%
        obstruction transitivity                                       31,157       0:00:03            0%
        short (length <= 4) crossing obstruction verification          52,722       0:00:02            0%
        point insertion                                                 1,181       0:00:02            0%
        fuse rows 1 and 2                                               2,359       0:00:02            0%
        fuse columns 1 and 2                                            2,339       0:00:02            0%
        fuse columns 2 and 3                                            2,409       0:00:02            0%
        fuse rows 2 and 3                                               2,428       0:00:02            0%
        locally factorable verification                                48,671       0:00:01            0%
        insertion encoding verified                                    48,686       0:00:00            0%
        One by one subclass of Av(01432)                               48,657       0:00:00            0%
        has specification                                              43,639       0:00:00            0%
        fuse rows 3 and 4                                               1,045       0:00:00            0%
        fuse columns 3 and 4                                            1,045       0:00:00            0%
        status                                                              6       0:00:00            0%
        fuse rows 0 and 1                                                   4       0:00:00            0%
        fuse columns 0 and 1                                                4       0:00:00            0%
    ClassDB status:
        Total number of combinatorial classes found is 313,142
        is_empty check applied 35586 time. Time spent: 0:00:02
    Queue status (currently on level 5):
        Queue              Size
        ---------------  ------
        working          23,517
        current (set 1)  39,996
        next                182
        The size of the current queues at each level: 1, 11, 111, 2091, 40014
    RuleDB status:
        Added from 1005796 normal rules
        Time spent computing forest keys: 0:08:15
        Time spent running the table method: 0:00:11
        Size of the gap: 3
        Size of the stable subset: 5522
        Sizes of the pre-images: [112107, 92240, 58509, 34463, 9141, 1160]

    Memory Status:
        ---------------------------------  ----------
        OS Allocated                         2.23 GiB
        Current Memory Used                2038.3 MiB
        Current Memory Allocated           2180.8 MiB
        Current JIT Memory Used              33.7 MiB
        Current JIT Memory Allocated         36.0 MiB
        Peak Memory Used                   2110.6 MiB
        Peak Memory Allocated Memory Used  2180.8 MiB
        ---------------------------------  ----------
        Total Garbage Collection Time: 0:03:32
     -- status update took 0.01 seconds --

[I 211122 11:53:30 comb_spec_searcher:468] Time taken so far: 1:07:20
    CSS status:
        Total time accounted for: 0:45:52
                                                                   Number of
                                                                 applications    Time spent    Percentage
        -----------------------------------------------------  --------------  ------------  ------------
        add assumptions                                               352,723       0:12:58           28%
        requirement placement strategy                                138,660       0:12:13           26%
        rearrange assumptions                                         352,723       0:09:06           19%
        requirement insertion                                          46,548       0:03:59            8%
        row and column separation                                      82,951       0:03:06            6%
        all placements                                                  1,181       0:01:50            4%
        factor                                                        121,877       0:00:58            2%
        requirement corroboration                                      25,011       0:00:56            2%
        tracked fusion                                                 25,011       0:00:12            0%
        verify atoms                                                   48,677       0:00:04            0%
        obstruction transitivity                                       31,170       0:00:03            0%
        short (length <= 4) crossing obstruction verification          54,360       0:00:02            0%
        point insertion                                                 1,181       0:00:02            0%
        fuse rows 1 and 2                                               2,367       0:00:02            0%
        fuse columns 1 and 2                                            2,339       0:00:02            0%
        fuse columns 2 and 3                                            2,409       0:00:02            0%
        fuse rows 2 and 3                                               2,436       0:00:02            0%
        fuse rows 3 and 4                                               2,865       0:00:02            0%
        locally factorable verification                                48,671       0:00:01            0%
        insertion encoding verified                                    48,686       0:00:00            0%
        One by one subclass of Av(01432)                               48,657       0:00:00            0%
        has specification                                              43,657       0:00:00            0%
        fuse columns 3 and 4                                            1,045       0:00:00            0%
        status                                                              7       0:00:00            0%
        fuse rows 0 and 1                                                   4       0:00:00            0%
        fuse columns 0 and 1                                                4       0:00:00            0%
    ClassDB status:
        Total number of combinatorial classes found is 500,580
        is_empty check applied 35586 time. Time spent: 0:00:02
    Queue status (currently on level 5):
        Queue               Size
        ---------------  -------
        working          449,272
        current (set 1)   39,996
        next                 232
        The size of the current queues at each level: 1, 11, 111, 2091, 40014
    RuleDB status:
        Added from 2051171 normal rules
        Time spent computing forest keys: 0:15:59
        Time spent running the table method: 0:00:19
        Size of the gap: 3
        Size of the stable subset: 9124
        Sizes of the pre-images: [205039, 171758, 69871, 34487, 9141, 1160]

    Memory Status:
        ---------------------------------  ----------
        OS Allocated                        3.605 GiB
        Current Memory Used                3241.1 MiB
        Current Memory Allocated           3481.8 MiB
        Current JIT Memory Used              34.0 MiB
        Current JIT Memory Allocated         36.0 MiB
        Peak Memory Used                   3482.3 MiB
        Peak Memory Allocated Memory Used  3548.2 MiB
        ---------------------------------  ----------
        Total Garbage Collection Time: 0:06:38
     -- status update took 0.01 seconds --
```

**Comment by jaypantone** [comment]
Finally got another status update:
```
[I 211122 14:59:56 comb_spec_searcher:468] Time taken so far: 4:13:47
    CSS status:
        Total time accounted for: 2:55:58
                                                                   Number of
                                                                 applications    Time spent    Percentage
        -----------------------------------------------------  --------------  ------------  ------------
        rearrange assumptions                                         649,155       1:08:27           38%
        add assumptions                                               649,155       1:00:54           34%
        requirement placement strategy                                286,606       0:29:00           16%
        row and column separation                                     139,881       0:08:02            4%
        requirement insertion                                          46,788       0:04:00            2%
        all placements                                                  1,181       0:01:50            1%
        factor                                                        184,640       0:01:33            0%
        requirement corroboration                                      25,013       0:00:56            0%
        fuse rows 1 and 2                                               9,879       0:00:13            0%
        tracked fusion                                                 25,013       0:00:12            0%
        fuse rows 2 and 3                                               9,948       0:00:12            0%
        fuse rows 3 and 4                                               8,546       0:00:09            0%
        verify atoms                                                   48,677       0:00:04            0%
        short (length <= 4) crossing obstruction verification          60,056       0:00:04            0%
        obstruction transitivity                                       31,170       0:00:03            0%
        point insertion                                                 1,181       0:00:02            0%
        fuse columns 1 and 2                                            2,339       0:00:02            0%
        fuse columns 2 and 3                                            2,409       0:00:02            0%
        locally factorable verification                                48,671       0:00:01            0%
        insertion encoding verified                                    48,686       0:00:00            0%
        One by one subclass of Av(01432)                               48,657       0:00:00            0%
        has specification                                              43,660       0:00:00            0%
        fuse columns 3 and 4                                            1,045       0:00:00            0%
        status                                                              8       0:00:00            0%
        fuse rows 0 and 1                                                   4       0:00:00            0%
        fuse columns 0 and 1                                                4       0:00:00            0%
    ClassDB status:
        Total number of combinatorial classes found is 1,023,712
        is_empty check applied 35586 time. Time spent: 0:00:02
    Queue status (currently on level 5):
        Queue                 Size
        ---------------  ---------
        working          2,675,580
        current (set 1)     39,996
        next                   234
        The size of the current queues at each level: 1, 11, 111, 2091, 40014
    RuleDB status:
        Added from 7403340 normal rules
        Time spent computing forest keys: 0:58:41
        Time spent running the table method: 0:01:03
        Size of the gap: 3
        Size of the stable subset: 19179
        Sizes of the pre-images: [438455, 393112, 128070, 34595, 9141, 1160]

    Memory Status:
        ---------------------------------  -----------
        OS Allocated                        10.495 GiB
        Current Memory Used                 8748.4 MiB
        Current Memory Allocated           10094.1 MiB
        Current JIT Memory Used               34.4 MiB
        Current JIT Memory Allocated          36.0 MiB
        Peak Memory Used                   10316.4 MiB
        Peak Memory Allocated Memory Used  10523.2 MiB
        ---------------------------------  -----------
        Total Garbage Collection Time: 0:25:39
     -- status update took 0.01 seconds --
```

**Comment by jaypantone** [comment]
Fixed by PR#400 which created a brand new tracked searcher.
